src/components/FooterContainer/Footer/Footer.js

```diff
@@ -11,14 +11,9 @@ import {
   EMAIL_ADDRESSES,
 } from '../../../constants';
 import Link from '../../Link/Link';
-import logo from '../../../images/australia-dta-logo.svg';
-import dtaLogo from './dtaLogo.svg';
-
+{ /* import logo from '../../../images/crest1.png';  */ }
```

Comment:
Rather delete than comment out, as the code can always be found again in the version history. Also delete associated resources if no longer required.